.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.0.2.cjs
```

Comment:
Not using `corepack` because it does not work quite well with Windows.

Reply:
Apparently there is some aversion to corepack as well 

https://github.com/nodejs/node/issues/50963#issuecomment-1885881236